scala/README.md

```diff
@@ -284,13 +327,28 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
     val answer = ys exists (...)
     ```
 
-0. Avoid wildcard-importing entire packages: `import package._`
+0. Avoid implicit conversions, use implicit value classes ([extension methods](http://docs.scala-lang.org/overviews/core/value-classes.html#extension-methods)) instead. In other words, do not import `scala.language.implicitConversions`:
```

Comment:
> In other words, do not import `scala.language.implicitConversions`:

to something like

> If the compiler tells you that you need to import `scala.language.implicitConversions`, that is a sign that you should change to use an implicit value class:

basically just trying to make it explicit that if you have to import that, you're doing it wrong (which i think is an easier way to remember it)